src/lib.rs

```diff
@@ -356,7 +350,6 @@ mod tests {
         env::set_var("WORKER_ID", "32");
         env::set_var("DATA_CENTER_ID", "0");
         env::remove_var("EPOCH");
-
```

Comment:
mm, i think these spaces are nice personally ^^; also the other tests have them, so either we remove them all or keep them. i'd prefer to keep!